libs/features/cdt/cdt.js

```diff
@@ -0,0 +1,124 @@
+import { getMetadata, getConfig, loadStyle } from '../../utils/utils.js';
+import { replaceKey } from '../placeholders.js';
+
+const replacePlaceholder = async (key) => replaceKey(key, getConfig());
+
+async function loadCountdownTimer(container) {
+  const cdtLabel = await replacePlaceholder('cdt-ends-in');
+  const cdtDays = await replacePlaceholder('cdt-days');
+  const cdtHours = await replacePlaceholder('cdt-hours');
+  const cdtMins = await replacePlaceholder('cdt-mins');
+  const cdtRange = (getMetadata('countdown-timer')).split(',');
```

Comment:
i would put this in initCDT and test existence of it then pass it to loadCountdownTimer, we can have mistakenly a block without a metadata